custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -52,6 +52,10 @@
 
 
 class BaseNikshayRepeater(CaseRepeater):
+    @property
+    def verify(self):
+        return True
```

Comment:
Did you mean to set this to `False`? I though the issue was that the certificate from Nikshay was failing the verification.

Reply:
Sorry. Yes..